jina/serve/runtimes/gateway/composite/gateway.py

```diff
@@ -21,10 +22,11 @@ def __init__(
         self.gateways: List[BaseGateway] = []
         for port, protocol in zip(self.ports, self.protocols):
             gateway_cls = _get_gateway_class(protocol)
-            runtime_args = copy.deepcopy(self.runtime_args)
+            # ignore metrics_registry since it is not copyable
+            runtime_args = deepcopy_with_ignore_attrs(self.runtime_args, ['metrics_registry'])
```

Comment:
I think we need to copy the runtime_args object, otherwise, how can different gateway objects hold the same reference of `runtime_args` but have different values of port and protocol ?